src/lib/FlightTasks/tasks/ManualAltitude/FlightTaskManualAltitude.cpp

```diff
@@ -85,13 +85,22 @@ bool FlightTaskManualAltitude::activate(vehicle_local_position_setpoint_s last_s
 
 void FlightTaskManualAltitude::_scaleSticks()
 {
-	// Use sticks input with deadzone and exponential curve for vertical velocity and yawspeed
-	_yawspeed_setpoint = _sticks_expo(3) * math::radians(_param_mpc_man_y_max.get());
+	// Use stick input with deadzone, exponential curve and first order lpf for yawspeed
+	const float yawspeed_target = _sticks_expo(3) * math::radians(_param_mpc_man_y_max.get());
+	_yawspeed_setpoint = _applyYawspeedFilter(yawspeed_target);
 
+	// Use sticks input with deadzone and exponential curve for vertical velocity
 	const float vel_max_z = (_sticks(2) > 0.0f) ? _constraints.speed_down : _constraints.speed_up;
 	_velocity_setpoint(2) = vel_max_z * _sticks_expo(2);
 }
 
+float FlightTaskManualAltitude::_applyYawspeedFilter(float yawspeed_target)
+{
+	const float den = math::max(_param_mpc_man_y_tau.get() + _deltatime, 0.01f);
+	const float alpha = _deltatime / den;
```

Comment:
How can you programmatically be sure alpha is always between 0 and 1?

Reply:
`_param_mpc_man_y_tau` is between 0 and inf, so `den` is between `_deltatime` and inf, so alpha is between 1 and 0.
The protection is just to protect against division-by-zero

Reply:
In the normal case, I was thinking of ways to break it with malicious input and what the effects are. Worst-case effect would be no yaw control or full speed yaw turning in erratic directions. But I think the case that I was thinking of is not possible with the max(...).

I think it's good.